FBRetainCycleDetector/Graph/FBObjectGraphConfiguration.h

```diff
@@ -49,7 +51,14 @@ typedef FBGraphEdgeType (^FBGraphEdgeFilterBlock)(FBObjectiveCGraphElement *_Nul
  */
 @property (nonatomic, readonly) BOOL shouldInspectTimers;
 
+/**
+ Will cache layout
+ */
+@property (nonatomic, readonly, nullable) NSMutableDictionary<Class, NSArray<id<FBObjectReference>> *> *layoutCache;
+@property (nonatomic, readonly) BOOL shouldCacheLayouts;
+
 - (nonnull instancetype)initWithFilterBlocks:(nonnull NSArray<FBGraphEdgeFilterBlock> *)filterBlocks
-                         shouldInspectTimers:(BOOL)shouldInspectTimers NS_DESIGNATED_INITIALIZER;
+                         shouldInspectTimers:(BOOL)shouldInspectTimers
+                           shouldCacheLayout:(BOOL)shouldCacheLayout NS_DESIGNATED_INITIALIZER;
```

Comment:
Hm.. I think my thinking there was to add an option for completeness, but that probably does not make sense. I at least cannot think of why we would pass NO there, so let me clean this up.